Rules/CommonScripts/ItemLimits.cfg

```diff
@@ -0,0 +1,101 @@
+#item limits list
+	
+	default					= 200
+
+	#structures
+	
+	tunnel					= 20
+	ballista              	= 20
+	warboat 				= 20
+	longboat				= 20
+	bomber					= 20
+	airship					= 20
+	dinghy					= 20
+	raft					= 20
+	catapult				= 30
+	saw						= 50
+	trampoline				= 50
+	tent					= 20
+	tdm_spawn				= 20
+	hall					= 20 # is disabled in online Sandbox
+	ctf_flag				= 20 # is disabled in online Sandbox
+	flag_base				= 20 # is disabled in online Sandbox
+	war_base				= 20 # is disabled in online Sandbox
+	
+	#animals
+	
+	fishy					= 50
+	chicken					= 50
+	bison					= 50 # is disabled in online Sandbox
+	shark					= 50 # is disabled in online Sandbox
+	
+	#others
+	
+	greg					= 20 # is disabled in online Sandbox
+	necromancer				= 20 # is disabled in online Sandbox
+	seed					= 50
+	tree_pine				= 50
+	tree_bushy				= 50
+	bush					= 50
+	flowers					= 50
+	grain_plant				= 50
+	
+	#items
+	
+	keg            			= 20
+	mine             		= 20
+	bucket					= 100
+	sponge					= 100
+	lantern					= 100
+	crate					= 100
+	boulder					= 100
+	drill					= 100
+	food					= 100
+	egg						= 100
+	
+	#unlimited - materials should rely on decay timers to disappear before they become a problem
+	
+	stone_door              = -1
+	wooden_door				= -1
+	bridge					= -1
+	ladder					= -1
+	wooden_platform			= -1
+	building				= -1
+	spikes					= -1
+	wire					= -1
+	elbow					= -1
+	tee						= -1
+	junction				= -1
+	diode					= -1
+	resistor				= -1
+	inverter				= -1
+	oscillator				= -1
+	transistor				= -1
+	toggle					= -1
+	randomizer				= -1
+	lever					= -1
+	push_button				= -1
+	coin_slot				= -1
+	pressure_plate			= -1
+	sensor					= -1
+	lamp					= -1
+	emitter					= -1
+	receiver				= -1
+	magazine				= -1
+	bolter					= -1
+	dispenser				= -1
+	obstructor				= -1
+	spike					= -1
+	mat_wood				= -1
+	mat_stone				= -1
+	mat_gold				= -1
+	mat_arrows				= -1
+	mat_waterarrows			= -1
+	mat_firearrows			= -1
+	mat_bombarrows			= -1
+	mat_bombs				= -1
+	mat_waterbombs			= -1
+	mat_bolts				= -1
+	mat_bomb_bolts			= -1
+	arrow					= -1
+	bomb					= -1
```

Comment:
The way it works is that when a blob isn't found, it will default to 200. But materials and builder blobs shouldn't be limited to 200 so these `-1` are added.
Of course it can be changed so there is no default and only stuff that is in the config is limited. Should that be done?

Reply:
I think it would make more sense for the default to be unlimited and anything specified here would override that default.

Reply:
As discussed on Discord, it would be smart to keep it the way it is to avoid the chance of someone forgetting to add a limit for new items.